appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: 2.0.0.{build}
-os: Visual Studio 2017 Preview
+os: Visual Studio 2017
```

Comment:
.NET Core SDK included in Visual Studio 2017 might not always be the one you want use. If instead you'd like to mimic travis behaviour and use some specific fixed version, feel free to take a look at **[my appveyor config](https://github.com/JustArchi/ArchiSteamFarm/blob/master/appveyor.yml#L19-L35)**, since it's very easy to install fixed SDK version and use it during build 👍.

Reply:
Typically an application should build with the latest SDK, but target for the lowest runtime it supports. AppVeyor currently ship SDK 2.0 on their VS2017 images, which suits me fine.

When there's a new SDK we can look at if we need to pin it - when 2.1 rolls around we can worry about it then. Hopefully they don't break compatibility 😄

Reply:
I think it's different for .NET Standard library then, `netcoreapp2.0` is very picky as SDK typically includes corefx libraries only for given version, so if they decide to bump it, even `2.1` can be fatal for me. This is what happens when you try to build `netcoreapp1.1` with 2.0 SDK:

```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '1.1.2' was not found.
  - Check application dependencies and target a framework version installed at:
      /
  - Alternatively, install the framework version '1.1.2'.
```

This is why in my case I prefer to pin it to 2.0, so suddenly the above won't happen with 2.1 SDK trying to build `netcoreapp2.0` 🙂.

Reply:
That would be because you don't have the 1.x runtime installed... I'm still not clear on what is supposed to be bundled with the SDK and what should be shipped separately. Hopefully that becomes well-defined by 2.1.

Reply:
> still not clear on what is supposed to be bundled with the SDK and what should be shipped separately.

Exactly, the only reason I decided to pin it, so CI won't break randomly at some point 😀. Still, probably unneeded for .NET standard library, but I decided to mention in case you'd be interested 👍.

Reply:
Our unit tests are netcoreapp.... 😈 

I'm OK in this instance with seeing how it breaks in CI first, rather than only finding out when someone upgrades VS/SDK.